docs/rfc/active/000-datadoc-entity/datadoc-entity-rfc.md

```diff
@@ -0,0 +1,72 @@
+- Start Date: (fill me in with today's date, 2022-02-22)
+- RFC PR: https://github.com/linkedin/datahub/pull/4237 
+- Discussion Issue: (GitHub issue this was discussed in before the RFC, if any)
+- Implementation PR(s): (leave this empty)
+
+# Extend data model to model Notebook entity 
+
+## Background
+[Querybook](https://www.querybook.org/) is Pinterest’s open-source big data IDE via a notebook interface. 
+We(Included Health) leverage it as our main querying tool. It has a feature, DataDoc, which organizes rich text, 
+queries, and charts into a notebook to easily document analyses. People could work collaboratively with others in a 
+DataDoc and get real-time updates. We believe it would be valuable to ingest the DataDoc metadata to Datahub and make
+it easily searchable and discoverable by others.
+
+## Summary
+This RFC proposes the data model used to model DataDoc entity. It does not talk about any architecture, API or other 
+implementation details. This RFC only includes minimum data model which could meet our initial goal. If the community 
+decides to adopt this new entity, further effort is needed.
+
+## Detailed design
+
+### DataDoc Model
+![DataDoc High Level Model](datadoc-high-level-model.png) 
```

Comment:
the filename that you commited is DataDoc-high-level-model.png --- could the casing be the issue?

Reply:
you may also need to prefix with `/.`

Reply:
you may also need to prefix with `/.`